resources/images/filters.go

```diff
@@ -174,6 +176,68 @@ func (*Filters) Padding(args ...any) gift.Filter {
 	}
 }
 
+// Dither creates a filter that dithers an image.
+func (*Filters) Dither(options ...any) gift.Filter {
+	type ditherOptions struct {
```

Comment:
An anonomous struct (ditherOptions := struct{}) would probably be a better choice here.